indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -146,6 +156,7 @@ public RemoteTaskRunner(
     this.workerPathCache = pathChildrenCacheFactory.make(cf, indexerZkConfig.getAnnouncementsPath());
     this.httpClient = httpClient;
     this.workerConfigRef = workerConfigRef;
+    this.exec = factory.create(1, "RemoteTaskRunner-Scheduled-Exec--%d");
```

Comment:
we probably don't need one executorservice per remotetaskrunner, one shared one should be enough?